packages/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -133,6 +186,18 @@ export class Tabs extends React.Component<TabsProps & OUIAProps, TabsState> {
     }
   };
 
+  insetString = (s: string) => (s === '2xl' ? '_2xl' : capitalizeFirstLetter(s));
```

Comment:
It's not clear what format is expected here. Some comments (i.e., examples) would help clarify how `capitalizeFirstLetter` is used to build modifiers.

Reply:
This is an internal function used only by this component  would a comment be enough or should I add examples to?

Reply:
Yes, just looking for code comments here